src/Numerical_aux/Functions.h

```diff
@@ -31,13 +31,22 @@
 #ifndef FUNCTIONS_20201026_H
 #define FUNCTIONS_20201026_H
 
+#include <cmath>
 #include <array>
 #include <cstddef>
 #include <cstdint>
 
 namespace seissol {
 namespace functions {
 
+/**
+ * @brief host standard math functions used in template metaprogramming
+ */
+struct HostStdFunctions {
+  template <typename T>
```

Comment:
Probably, we need a `#pragma omp declare simd` here to enable vectorization.

Reply:
It seems to me that there is not way to add `#pragma omp declare simd` to a templated function

Reply:
Here is the code from the master branch